src/main/java/org/quantumbadger/redreader/activities/MainActivity.java

```diff
@@ -631,6 +722,74 @@ protected void doRefresh(final RefreshableFragment which, final boolean force, f
 		invalidateOptionsMenu();
 	}
 
+	private void setupDrawerLayout(View layout, RedditAccount redditAccount) {
+		if (!redditAccount.isAnonymous()) {
+			userInterface = layout.findViewById(R.id.ll_user_interface);
+			TextView userTextView = layout.findViewById(R.id.tv_username);
+			ImageButton mailedButton = layout.findViewById(R.id.btn_mailed);
+			ImageButton savedButton = layout.findViewById(R.id.btn_saved);
+			ImageButton hiddenButton = layout.findViewById(R.id.btn_hidden);
+			ImageButton upVotedButton = layout.findViewById(R.id.btn_upvoted);
+			ImageView profileImage = layout.findViewById(R.id.iv_profile);
+			profileImage.setVisibility(View.VISIBLE);
+			Glide.with(this).load(redditAccount.profileImageUrl).into(profileImage);
+			userInterface.setVisibility(View.VISIBLE);
+			userTextView.setText(redditAccount.username);
+			setupUserInterfaceClickListener(profileImage, mailedButton, savedButton, hiddenButton, upVotedButton);
+		}
+		mTabLayout = layout.findViewById(R.id.tabLayout);
+		mDrawer = layout.findViewById(R.id.drawer_layout);
+		setupTabListener();
+		mLeftPane = (FrameLayout) layout.findViewById(R.id.main_left_frame);
+		postListingController = new PostListingController((PostListingURL) RedditURLParser.parseProbablePostListing(SubredditPostListURL.getFrontPage().generateJsonUri()), this);
+		postListingController.setSort(mOrder);
+		postListingFragment = postListingController.get(this, true, null);
+		mLeftPane.addView(postListingFragment.getView());
+	}
+
+	private void setupUserInterfaceClickListener(ImageView profileImage,
+												 ImageButton mailedButton,
```

Comment:
Indentation is a bit off -- better to put `profileImage` on a new line, and just indent all the parameters two tabs (relative to the method signature).